pkg/runctl/cfg/config.go

```diff
@@ -151,7 +151,7 @@ func processNetworkMap(networkMap map[string]string, config *PipelineRunsConfigS
 
 	networkPolicies := map[string]string{}
 	for key, value := range networkMap {
-		if key != defaultNetworkPolicyKey && key != "" && !strings.HasPrefix(key, "_") {
+		if key != defaultNetworkPolicyKey && key != "" && strings.TrimSpace(value) != "" && !strings.HasPrefix(key, "_") {
```

Comment:
The same checks must be applied to `defaultNetworkPolicyKey` (and the respective value), too.

Reply:
I think we want to allow empty defaultNetworkPolicy to apply no additional network policy.
See run_manager.go#L381

Reply:
See https://github.com/SAP/stewardci-core/pull/160/commits/2d9adfb6aa51bbf712c1ae99c562292afb8a28fc